src/service/basin_subzone.js

```diff
@@ -216,6 +216,25 @@ module.exports = (basinSubzonePersistence, seService) => {
       };
     },
 
+    /**
+     * Request a given strategic ecosystem HF timeline data inside a basin subzone
+     * @param {Number} subzoneId basin subzone id
+     * @param {String} seType strategic ecosystem type
+     *
+     * @return {Object} Object of HF values through time
+     */
+
+    getSEHFTimeline: async (eaId, seType) => {
```

Comment:
the first parameter should be called `subzoneId`

Reply:
Done